content/en/guides/hosting/data-security/presigned-urls.md

```diff
@@ -24,36 +25,40 @@ A pre-signed URL expires after:
 
 ## Team-level access control
 
-Each pre-signed URL is restricted to specific buckets based on [team level access control]({{< relref "/guides/hosting/iam/access-management/manage-organization.md#add-and-manage-teams" >}}) in the W&B platform. If a user is part of a team which is mapped to a blob storage bucket using [secure storage connector]({{< relref "./secure-storage-connector.md" >}}), and if that user is part of only that team, then the pre-signed URLs generated for their requests would not have permissions to access blob storage buckets mapped to other teams. 
+Each pre-signed URL is restricted to specific buckets based on [team level access control]({{< relref "/guides/hosting/iam/access-management/manage-organization.md#add-and-manage-teams" >}}) in the W&B platform. If a user is part of a team which is mapped to a storage bucket using [secure storage connector]({{< relref "./secure-storage-connector.md" >}}), and if that user is part of only that team, then the pre-signed URLs generated for their requests would not have permissions to access storage buckets mapped to other teams. 
 
 {{% alert %}}
 W&B recommends adding users to only the teams that they are supposed to be a part of.
 {{% /alert %}}
 
 ## Network restriction
+W&B recommends using IAM policies to restrict the networks that can use pre-signed URLs to access external storage using pre-signed URLs. This helps to ensure that your W&B specific buckets are accessed only from networks where your AI workloads are running, or from gateway IP addresses that map to your user machines. 
 
-W&B recommends restricting the networks that can use pre-signed URLs to access the blob storage, by using IAM policy based restrictions on the buckets. 
-
-In case of AWS, one can use [VPC or IP address based network restriction](https://docs.aws.amazon.com/AmazonS3/latest/userguide/using-presigned-url.html#PresignedUrlUploadObject-LimitCapabilities). It ensures that your W&B specific buckets are accessed only from networks where your AI workloads are running, or from gateway IP addresses that map to your user machines if your users access artifacts using the W&B UI.
+In CoreWeave, AWS S3, or S3-compatible storage, you can define network restrictions at the VPC or IP address level. Refer to:
```

Comment:
I'm not sure this works for coreweave, I don't think we should include coreweave here unless we know it works

Reply:
It looks like maybe not for VPC. I looked here:  https://docs.coreweave.com/docs/products/storage/object-storage/reference/bucket-policy#condition WDYT?